pandapower/converter/jao/from_jao.py

```diff
@@ -0,0 +1,987 @@
+    """Creates buses to the pandapower net using information from the lines and tielines sheets
+    (excel file).
+
+    Parameters
+    ----------
+    net : pandapowerNet
+        net to be filled by buses
+    data : dict[str, pd.DataFrame]
+        data provided by the excel file which will be corrected
+    """
+    bus_df_empty = pd.DataFrame({"name": str(), "vn_kv": float(), "TSO": str()}, index=[])
+    bus_df = deepcopy(bus_df_empty)
+    for key in ["Lines", "Tielines"]:
+        for subst in ['Substation_1', 'Substation_2']:
+            data_col_tuples = [(subst, "Full_name"), (None, "Voltage_level(kV)"), (None, "TSO")]
+            to_add = data[key].loc[:, data_col_tuples].set_axis(bus_df.columns, axis="columns")
+            if len(bus_df):
+                bus_df = pd.concat([bus_df, to_add])
+            else:
+                bus_df = to_add
+    bus_df = _drop_duplicates_and_join_TSO(bus_df)
+    new_bus_idx = create_buses(
+        net, len(bus_df), vn_kv=bus_df.vn_kv, name=bus_df.name, zone=bus_df.TSO)
+    assert np.allclose(new_bus_idx, bus_df.index)
+
+
+def _create_lines(net:pandapowerNet, data:dict[str, pd.DataFrame]) -> None:
+    """Creates lines to the pandapower net using information from the lines and tielines sheets
+    (excel file).
+
+    Parameters
+    ----------
+    net : pandapowerNet
+        net to be filled by buses
+    data : dict[str, pd.DataFrame]
+        data provided by the excel file which will be corrected
+    """
+
+    bus_idx = _get_bus_idx(net)
+
+    for key in ["Lines", "Tielines"]:
+        length_km = data[key][("Electrical Parameters", "Length_(km)")].values
+        zero_length = np.isclose(length_km, 0)
+        no_length = np.isnan(length_km)
+        if sum(zero_length) or sum(no_length):
+            logger.warning(f"According to given data, {sum(zero_length)} {key.lower()} have zero "
+                           f"length and {sum(zero_length)} {key.lower()} have no length data. "
+                           "Both types of wrong data are replaced by 1 km.")
+            length_km[zero_length | no_length] = 1
+        vn_kvs = data[key].loc[:, (None, "Voltage_level(kV)")].values
+
+        _ = create_lines_from_parameters(
+            net,
+            bus_idx.loc[list(tuple(zip(data[key].loc[:, ("Substation_1", "Full_name")].values,
+                        vn_kvs)))].values,
+            bus_idx.loc[list(tuple(zip(data[key].loc[:, ("Substation_2", "Full_name")].values,
+                        vn_kvs)))].values,
+            length_km,
+            data[key][("Electrical Parameters", "Resistance_R(Ω)")].values / length_km,
+            data[key][("Electrical Parameters", "Reactance_X(Ω)")].values / length_km,
+            data[key][("Electrical Parameters", "Susceptance_B(μS)")].values / length_km,
+            data[key][("Maximum Current Imax (A)", "Fixed")].fillna(999000).values / 1e3,
```

Comment:
as mentioned above, dummy max_I should not be set silently and without explicit permission from the user